sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/GroupIntoBatches.java

```diff
@@ -117,6 +117,11 @@
    */
   @AutoValue
   public abstract static class BatchingParams<InputT> implements Serializable {
```

Comment:
It looks like your adding support for GroupIntoBatches to limit on count and byte size at the same time.

Can you add tests that cover this new scenario to:
* GroupIntoBatchesTest
* GroupIntoBatchesTranslationTest

Reply:
see [`040b744` (#22953)](https://github.com/apache/beam/pull/22953/commits/040b744454a6985ca2bb1c42e30fe90fabd42a59) and [`b1b732c` (#22953)](https://github.com/apache/beam/pull/22953/commits/b1b732c60f2dbeebfddc50afa7a946f2ed0864ca)